components/components.yaml

```diff
@@ -823,3 +823,12 @@
   ProjectUrl: https://github.com/ParticularLabs/NServiceBus.AwsLambda
   NugetOrder:
     - NServiceBus.AwsLambda.SQS
+
+- Key: Microsoft.Hosting
+  Name: Microsoft.Hosting
+  DocoUrl: nservicebus/hosting/microsoft-extensions
```

Comment:
I think this shouldn't be on the `extensions-hosting` URL as there are many Microsoft.Extensions.* packages. How about `extensions-hosting`?